xil_carma_cloud/docker-compose.yml

```diff
@@ -249,7 +254,11 @@ services:
       - /opt/carma/maps:/opt/carma/maps
       - /opt/carma/routes:/opt/carma/routes3
       - /opt/carma/yolo:/opt/carma/yolo
-    command: bash -c 'wait-for-it.sh 172.5.0.2:11311 -- rosparam load /opt/carma/vehicle/config/bridge.yml && source ~/.base-image/workspace/install/setup.bash && ros2 run ros1_bridge dynamic_bridge --multi-threads'
+    command: >
+      bash -c 'export RMW_IMPLEMENTATION=rmw_fastrtps_cpp
```

Comment:
Is this change resource specific. Also do we want to pull this in as a docker compose environment variable since it problably is set  accross all ros images/services right.

Reply:
Yeah `rmw_fastrtps_cpp` is resource specific, but we want to keep this change to be consistent. 
The AWS instance may not need `rmw_fastrtps_cpp` but instead might work better with `rmw_cyclonedds_cpp` which you can change it in your own PR.